content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/count_products.rst

```diff
@@ -43,7 +43,7 @@ with each product line containing the following information:
   :guilabel:`Counted Quantity`, once an inventory adjustment is made. The difference is
   automatically calculated after every inventory adjustment.
 - :guilabel:`Scheduled Date`: the date at which a count should be made. If not otherwise specified,
-  this date will default to the 31st of December of the current year.
+  this date defaults to the 31st of December of the current year.
```

Comment:
commenting here since I can't comment on line 69, but in 16.0+, the "Create" button is changed to "New"
![image](https://github.com/user-attachments/assets/1d9fa3a2-09a6-47b3-ae21-0dbc04550705)

Also, we have yet to mention the "Relocate" button in this doc (it's new in 17.0!). This is also my bad, as I forgot to mention it in the task. 

The relocate button is a quick and easy way to move objects to another location. It happens when products end up in unexpected places and we need to record where it's actually at, or where it's supposed to be stored. Let me know if you have any additional questions about the Relocate feature! I'll let you decide where it's best to put this section (:

Reply:
In line 61, there's a reference of the "Additional options" button. We no longer call it that, and instead just use :icon:`oi-settings-adjust` :guilabel:`(settings)` icon.

Reply:
In line 109, we mention the "Apply button" at the far right of the page. While it's correct, I'd like you to modify it in your edit to include the icon. (: 
![image](https://github.com/user-attachments/assets/e7036717-7b94-4fa0-9a6d-2da5636fd098)